specs/VersionInfo/VersionInfo.md

```diff
@@ -0,0 +1,188 @@
+satisfies the dependency which is >= MinVersion (not ==MinVersion).
+
+One can indirectly determine the actual runtime version in use by retrieving the package graph via
+[GetCurrentPackageInfo()](https://docs.microsoft.com/windows/win32/api/appmodel/nf-appmodel-getcurrentpackageinfo)
+and walking the result looking for a framework package in the runtime's family. This is cumbersome
+(even more so for C# given the need for P/Invoke and the shape of the API for C# callers), can pose
+performance overhead (searching across a deep copy of the package graph) and is potentially
+ambiguous (e.g. advanced scenarios using Dynamic Dependencies can have multiple runtimes in the
+package graph).
+
+This can be shrinkwrapped via an easy to use API where we (WinAppSDK) incur the burden of
+implementation complexity, using a design to directly and efficiently yield the correct answer.
+
+This provides a simple but reliable and performant solution for applications, as well as class
+libraries and other components using WinAppSDK.
+
+# Examples
+
+An example using the runtime version information API to display the actual version in use at runtime:
+
+```c#
+using System;
+using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel.WindowsAppRuntime;
+
+public class MyInfo
+{
+    public static void About()
+    {
+        var build_release = Microsoft.WindowsAppSDK.Release;
+        var build_version = Microsoft.WindowsAppSDK.Runtime.Version;
+        var release = VersionInfo.ReleaseInfo.Current();
+        var runtime = .VersionInfo.RuntimeInfo.Current();
+        var message = $"Built with Windows App SDK ${build_release.Major}.${build_release.Minor}.${build_release.Patch}${build_release.FormattedVersionTag}\n"
+                      $"Minimum Windows App Runtime version ${build_version.Major}.${build_version.Minor}.${build_version.Build}.${build_version.Revision}\n"
+                      "\n" +
+                      $"Running with Windows App SDK ${release.Major}.${release.Minor}.${release.Patch}${release.FormattedVersionTag}\n" +
+                      $"Windows App Runtime version ${runtime.version.Major}.${runtime.version.Minor}.${runtime.version.Build}.${runtime.version.Revision}";
+        MessageBox.Show(message, "About MyApp", MessageBoxButtons.OK, MessageBoxIcon.Information);
+    }
+}
+```
+
+# Remarks
+
+Version information is compiled into `Microsoft.WindowsAppRuntime.Insights.Resource.dll` as the
+product is assembled (i.e. when the version is defined). The VersionInfo API retrieves this
+information via an internal API.
+
+# 6. API Details
+
+```c# (but really MIDL3)
+namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime.VersionInfo
+{
+    [contractversion(1)]
+    apicontract VersionInfoContract{};
+
+    /// Version information for the Windows App SDK release.
+    [contract(VersionInfoContract, 1)]
+    runtimeclass ReleaseInfo : Windows.Foundation.IStringable
```

Comment:
Is there a need for this to be a seperate RTC instance? Feels like it's a lot of methods for retrieving three UInt16s and two strings.

Reply:
DISCUSSION: Yes, 2 separate RTC because 2 different contexts of information.

RECOMMEND: Drop .Current and make all methods static. No need for instances